### DISCUSSION TITLE
【灌水】好无聊。。。有没有来玩mc联机的绳匠

### DISCUSSION THREAD

**Discussion by Pinuttt** [share121/inter-knot#2393] [其他]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/01c96af07865b0d5215c9bc97ebeca73_8859049719826705182.png" alt="undefined" width="75" >摸鱼时间到！

**Comment by Pinuttt** [comment]
别问，问就是有人再开房间，我是网易低等玩家，，

**Comment by Jshuya24** [comment]
网易还是算了吧

**Comment by maopao111** [comment]
pcl2可以一起吗<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >
